[2022-06-18 12:55:21,018] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts-Papple-iphone-xs-256gb-space-grau_h_6245 scheduled__2022-06-18T10:22:20.195105+00:00 [queued]>
[2022-06-18 12:55:21,620] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts-Papple-iphone-xs-256gb-space-grau_h_6245 scheduled__2022-06-18T10:22:20.195105+00:00 [queued]>
[2022-06-18 12:55:21,759] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 12:55:21,759] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-06-18 12:55:21,759] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 12:57:28,311] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): pts-Papple-iphone-xs-256gb-space-grau_h_6245> on 2022-06-18 10:22:20.195105+00:00
[2022-06-18 12:57:29,903] {standard_task_runner.py:52} INFO - Started process 5269 to run task
[2022-06-18 12:57:31,467] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Job-pts', 'pts-Papple-iphone-xs-256gb-space-grau_h_6245', 'scheduled__2022-06-18T10:22:20.195105+00:00', '--job-id', '437', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/price_scale.py', '--cfg-path', '/tmp/tmpvb9ygf3t', '--error-file', '/tmp/tmpwkdl9gbx']
[2022-06-18 12:57:31,486] {standard_task_runner.py:80} INFO - Job 437: Subtask pts-Papple-iphone-xs-256gb-space-grau_h_6245
[2022-06-18 12:57:39,712] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/configuration.py:525: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-06-18 12:59:02,444] {standard_task_runner.py:97} ERROR - Failed to execute job 437 for task pts-Papple-iphone-xs-256gb-space-grau_h_6245 ((psycopg2.OperationalError) could not translate host name "postgres_source" to address: Temporary failure in name resolution

(Background on this error at: http://sqlalche.me/e/14/e3q8); 5269)
[2022-06-18 12:59:08,345] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-18 13:01:52,190] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Job-pts, task_id=pts-Papple-iphone-xs-256gb-space-grau_h_6245, execution_date=20220618T102220, start_date=20220618T125521, end_date=20220618T130151
[2022-06-18 13:01:58,544] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
